### ISSUE TITLE
Redundant hyperlinks Impact Accessibility

### ISSUE THREAD

**Issue by nehagupta06** [w3c/wcag#2637]
**Issue Description:**
Redundant hyperlinks are a common accessibility issue. They’re also a potential violation of the Americans with Disabilities Act (ADA). A link is redundant if it’s next to another hyperlink that leads to the same address. This often occurs when websites use images that act as hyperlinks alongside text that also acts as hyperlinks. 

**Why is this a problem?** 
Redundant links are frustrating for users, the best practice is to fix your content by combining both elements in the same hyperlink.

People who navigate with a keyboard will need to navigate through both elements to move past the hyperlink. Screen reader users may hear the hyperlink announced twice (once for the icon hyperlink and a second time for the text). 

**An Example of a Redundant Link**
Let’s say we navigate inside a Product listing page of an e-commerce website that has a product image and its corresponding name that appears alongside the product. Both of those elements have hyperlinks that lead to the same page i.e. to the Product description page.

**We can fix this issue in HTML with two basic techniques:** 
Wrap both elements in the same link (or an element). 
Provide a null text alternative for the image, along with accurate link text.

**Comment by patrickhlauke** [comment]
> They’re also a potential violation of the Americans with Disabilities Act (ADA)

doubtful.

not having redundant links is certainly a best practice, but it wouldn't directly fail any WCAG success criteria

also, why are you posting this here as an issue? sounds more like you're trying to post a statement?

**Comment by bruce-usab** [comment]
@nehagupta06 please close this issue and use the Discussions tab to start a conversation there.  Please ping from your new thread.   I have been meaning to share some newer resources from DOJ and ED.

**Comment by fstrr** [comment]
This issue hasn't been updated since August 2022 and looks like it can be closed. If further discussion needs to happen, please open a Discussion.
